tictactoe/test/game_loop_tests.erl

```diff
@@ -0,0 +1,12 @@
+-module(game_loop_tests).
+-include_lib("eunit/include/eunit.hrl").
+
+
+complete_turn_test() -> 
+  [{  "completes a turn by either starting a new turn or ending the game when won or board full",
```

Comment:
Ok my biggest gripe here is that, like the IO methods, you are not actually testing the result of the method here, only that the method executed. I see what you are trying to do, but it would make more sense if there was some sort of output from the `complete_turn` method, like a "string" or something that you could compare against. 

I am going to have to defer to someone who knows more than me, though, because I am not doing a good job of explaining this. This is another "pass-for-now" to keep it moving, but this needs work.

Reply:
This actually does do what you are saying, I'll elaborate...
These expected results really say, result = displayed game end message ("string"), they are not evaluated in the shell but rather in a background process since they are being run inside an eunit test:
`console_io:display(utilities:o_wins())` -> expect display of "Player 1, the 'x' player has won the game.\n".
`console_io:display(utilities:x_wins()`
`console_io:display("Draw!\n")`
These say, evaluate (in the background) the complete turn function on this mocked board:
`game_loop:complete_turn([o,2,3,o,5,6,o,8,9])` -> Board passed in has been won by O
`game_loop:complete_turn([x,2,3,4,x,6,7,8,x]))` -> Board passed in has been won by X
`game_loop:complete_turn([x,o,x,o,x,x,o,x,o]))`  -> Board passed in is drawn
Then the two arguments returned values are compared for equality in the background and eunit displays the diff between the return of the evaluation of the expected result (displayed strings) and the return of the evaluation of the function to test (complete turn w/ mocked board)...
All that being said: I am missing a  test case that proves another turn is executed when a board that is not drawn or won is evaluated.  I had some trouble with this I think because of  the fact that processes started by prompts encountered in the take_turn method terminate without passing on a value when run in the background and as such result in an error, I believe I know a way to design around this and will update the design with a branch specifically to update testing and fix design shortcomings after I've finished working on minimax